content/installation/java/Java-basic-installation.md

```diff
@@ -6,16 +6,16 @@ tags: "installation Java agent overview installation"
 
 To install the Java agent, complete the following steps:
 
-* Log in to the Contrast interface.
+* Log in to the Contrast UI.
 * Click the button in the top navigation bar to **Add Agent**.
 * Select the **Java** agent in the dropdown menu, and click the button to **Download Agent**. You may need to specify proxy authentication information required by your network before downloading the agent.
 
 <a href="assets/images/Download-java-agent.png" rel="lightbox" title="Download the Java agent"><img class="thumbnail" src="assets/images/Download-java-agent.png"/></a>
 
+* After download, add a JVM parameter to install the Java agent. 
 
-> **Note:** You must not rename the *contrast.jar* file. If the agent file name is changed, the agent will [fail to connect](troubleshooting-javainstall.html#class) with the Contrast application. 
```

Comment:
This change deletes an important note for our customers. Regardless of where a customer retrieves the agent jar from, it is still the case that the customer cannot rename this file. We are redesigning a core part of the agent that will soon eliminate this restriction, but for now it is a common source of support issues